source/math/FloatVectorList.ooc

```diff
@@ -432,9 +433,51 @@ FloatVectorList: class extends VectorList<Float> {
 		}
 		(crossCorrelation, offsets)
 	}
+	// Assumes equi-distant samples
+	// https://en.wikipedia.org/wiki/Spline_interpolation
+	interpolateSpline: func (numberOfPoints: Int) -> This {
+		result := This new(this count + numberOfPoints * (this count - 1))
+		thisPointer := this pointer as Float*
+		coefficientMatrix := This _generateCubicSplineCoefficients(this count)
+		b := FloatMatrix new(1, this count) take()
+		for (index in 0 .. this count)
+			b[0, index] = 3.f * (thisPointer[Int minimum(this count - 1, index + 1)] - thisPointer[Int maximum(0, index - 1)])
+		// TODO: Since coefficientMatrix is tri-diagonal, there are better methods for this. Both in terms of data storage and solving.
```

Comment:
Create an issue for this. Either PR debt or Later depending on how important it is.